lib/deep-equal.js

```diff
@@ -226,6 +226,36 @@ internals.isDeepEqualObj = function (instanceType, obj, ref, options, seen) {
         }
     }
 
+    // Check symbols
+
+    const objSymbols = getOwnPropertySymbols(obj);
+    const refSymbols = new Set(getOwnPropertySymbols(ref));
+
+    for (let i = 0; i < objSymbols.length; ++i) {
```

Comment:
Use for...of in the future.

Reply:
Will do. Maybe we should update the style guide?

Reply:
Yep. @cjihrig ...